next.config.js

```diff
@@ -1,8 +1,14 @@
-module.exports = {
-  future: {
+/**
+ * @type {import('next').NextConfig}
+ */
+ const nextConfig = {
+  env: {
```

Comment:
Thanks for the quick changes @ranitasahaa1230. Adding webpack5 inside the env object solves the issue. This could have been the second option if we had no other work around. But I guess, now the webpack 5 is default for all NextJS apps. Please check the linked blog
Also add **Closes** keyword before your issue number, so that the ticket is closed automatically once the PR is merged 
https://nextjs.org/docs/messages/future-webpack5-moved-to-webpack5

Reply:
Thank you. I have made the changes. Please review it.